app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -40,257 +39,273 @@
 
             <!-- Placeholder. Height gets set at runtime based on container size; the initial value is a hack to keep
                  the detail info offscreen until it's placed properly. May be a better way to do this. -->
-            <com.facebook.drawee.view.SimpleDraweeView
-                android:id="@+id/mediaDetailImageView"
+
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/dimen_250"
-                app:actualImageScaleType="none" />
+                android:orientation="vertical"
+                android:background="@android:color/transparent"
+                android:id="@+id/mediaDetailImageViewSpacer"
+                />
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="?attr/fragmentCategorisationBackground"
-                android:orientation="vertical"
-                android:padding="@dimen/standard_gap">
+                android:background="?attr/mainBackground"
+                android:orientation="vertical">
 
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:background="?attr/subBackground"
-                    android:orientation="vertical"
-                    android:padding="@dimen/standard_gap">
+                    android:weightSum="100"
```

Comment:
I personally really hate the `weightsum` attribute. You can omit it if you have specified the weight of every child view, using weightsum should be to change the default behaviour of not specifying the weight of a view, which is a bad practice anyway, just specify the weight of every child if you are positioning them proportionately.

This is of course all legacy knowledge, best practice is use constraintlayout everywhere* and probably in a years time use Compose everywhere.

Reply:
thanks, it didn't need weightsum eventually :)